manim/mobject/svg/brace.py

```diff
@@ -119,14 +123,17 @@ def get_tex(self, *tex, **kwargs):
 
     def get_tip(self):
         # Returns the position of the seventh point in the path, which is the tip.
-        return self.points[28]  # = 7*4
+        if config["renderer"] == "opengl":
+            return self.get_points()[34]
```

Comment:
A comment for where 34 comes from would be great.

Reply:
Actually I have no idea about this. I had to manually test the number that returns the center of the tip, and 34 is that number.

Reply:
I've opened an issue for this.